src/scrabble.js

```diff
@@ -1,5 +1,58 @@
-function scrabble() {
+function scrabble(scrabbleWord) {
+  if (!scrabbleWord) return 0
+  const libary = {
```

Comment:
Might be a good idea to define library outside of the scope of scrabble()

Reply:
could you please explain why ?

Reply:
My thinking is that this way only your scrabble() function has access to the library. If you were to declare separate functions for the extensions besides scrabble(), like, let's say, tripleLetter(), and then call it inside scrabble(), tripleLetter() wouldn't be able to use your library because your library lives only inside scrabble(). For now, the way your code works it's completely fine though, that's why I was saying "might be" :)

Reply:
thank you for the explanation. That is a very good point. essentially my current code will then be a bit more limited than if it was done the otherway.
thank you